dayrepo/snippets/urls.py

```diff
@@ -17,4 +17,5 @@
     path("delete/<int:target_id>/<str:delete_type>/", views.db_delete,name="db_delete"),
     path("excel/<int:snippet_pk>/", views.excelfile_download,name="excelfile_download"),
     path("cars/", master_views.car_list,name="car_list"),
+    path("add/cars/", master_views.car_add,name="car_add"),
```

Comment:
パスが
snippets/add/cars
となっており、　snippetsをaddしていそうなのと、複数のcarが関わるような命名になっています。

他の実装に合わせるならnewになるのではないでしょうか。
```suggestion
    path("car/new/", master_views.car_add,name="car_add"),
```

Reply:
こちらで対応しました
https://github.com/bob-g12/dayrepo-for-transportation/pull/70/commits/1ee91ed6317131b7b4407d16c0d29ceb89785af0